moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -636,6 +636,24 @@ TEST_F(OneRobot, testInterpolation)
     EXPECT_NEAR(1 - t, interpolated_state.getVariablePosition("joint_f"), 1e-9);
     EXPECT_NEAR(1.5 * (1 - t) + 0.1, interpolated_state.getVariablePosition("mim_f"), 1e-9);
   }
+
+  std::cout << "Checking interpolation input verification. Two warnings (untested) and an exception are expected."
```

Comment:
well, "untested" really doesn't make any sense in a test. :-)
Also the exception is not visible on the command line (you catch it), so there's no need to mention it at all.

Reply:
What I'm saying is that lines 664 and 665 _should_ produce warnings, but I didn't actually add the `/rosout_agg` listener to verify that they do. So it's code that's in a test but if it doesn't behave properly, the test will still pass (maybe not the best approach, now that I think about it).

As for the exception, I do catch it but I also print out the error message.

Reply:
Ah, sorry for the confusion. Your explanations make a lot of sense.
"untested" is a word you really don't want to see in tests. :-)

Testing extrapolation can make sense, but testing for the warnings does not seem too useful in my opinion.

Thanks for adding the clarification on the cout output as well.

I'll go ahead and merge.